software/motion-tools.yml

```
@@ -0,0 +1,12 @@
+name: "motion.tools (Antragsgrün)"
+website_url: "https://motion.tools/"
+source_code_url: "https://github.com/CatoTH/antragsgruen"
+description: "A web tool for managing motions and amendments for (political) conventions"
```

```suggestion
description: "A web tool for managing motions and amendments for (political) conventions".
```
Just to add a dot at the end

---

`.` needs to come before the `"`

---

Updated, thank you 👍🏻